olm/prepare-community-operators-update.sh

```diff
@@ -132,38 +135,29 @@ do
     echo $platform
     echo "     => will create release '${lastPackagePreReleaseVersion}' in the following package folder :'${folderToUpdate}'"
 
-    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}"
+    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}/{manifests,metadata}"
+    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}/metadata"
```

Comment:
I think we can remove this line, cause we've created metadata folder in the line upper.